Amazon.IonDotnet.Tests/Internals/ReaderLocalTableTest.cs

```diff
@@ -16,23 +16,20 @@
 using System;
 using Amazon.IonDotnet.Builders;
 using Amazon.IonDotnet.Internals;
-using Amazon.IonDotnet.Tree;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 namespace Amazon.IonDotnet.Tests.Internals
 {
     [TestClass]
     public class ReaderLocalTableTest
     {
-        internal const int UNKNOWN_SYMBOL_ID = -1;
-        internal static string ION_SYMBOL_TABLE = "$ion_symbol_table";
-        internal static string LocalSymbolTablePrefix = ION_SYMBOL_TABLE + "::";
+        internal static string LocalSymbolTablePrefix = SystemSymbols.IonSymbolTable + "::";
         internal static string ION_1_0 = "$ion_1_0";
```

Comment:
Sorry, I missed this. This can also be replaced by SystemSymbols.Ion10.

Reply:
Updated